.github/CODEOWNERS

```diff
@@ -1070,9 +1070,9 @@
 # ServiceOwners:                                                   @sunkun99
 
 # PRLabel: %Compute - Fleet
-/sdk/computefleet/Azure.ResourceManager.*/                         @sahilarora92 @dhruvil009 @rahuls-microsoft
+/sdk/computefleet/                                                 @sahilarora92 @dhruvil009 @rahuls-microsoft
```

Comment:
Please revert this file.   It was properly updated in a separate PR.

Reply:
Restored to head - git restore -s HEAD .github\CODEOWNERS

Reply:
Nope.  Still showing changes.   This file should not be included in the change set.

Reply:
Ok, what command should be used?

Reply:
I used -
git remote add mainSource https://github.com/rahuls-microsoft/azure-sdk-for-net.git
git fetch mainSource
git checkout main
git reset --hard mainSource/main  
git push origin main --force 

Is that fine?
How do I check it, as you checked? So, I can validate it before resolving it.

Reply:
I think the easiest way to revert is
- pull latest main to your local
- then run "git checkout origin/main .github/CODEOWNERS" under the repo root folder

Reply:
Also execute - git rebase mainSource/main - This should resolve the issue.
Else please let me know. Thanks

Reply:
> I think the easiest way to revert is
> 
> * fetch/pull latest origin/main to your local
> * switch to your working branch
> * then run "git checkout origin/main .github/CODEOWNERS" under the repo root folder

I have created a fork - https://github.com/rahuls-microsoft/azure-sdk-for-net/.
SO, taking too long time and showing lots of conflicts. Is there an easier way or should I just create branch from - https://github.com/Azure/azure-sdk-for-net. That may be an easier option.
Please suggest. Thanks for your help!

Reply:
I reverted it for you

Reply:
Thanks

Reply:
Sorry! How do I undo it?

Reply:
I redid it.
Merge the latest from remote before you push to remote, instead of force push, it is error-prune with force push.

Reply:
@live1206 - Please apply the changes. Thanks for yourr help.